packages/config/config/devices/0x015f/mh-s511.json

```diff
@@ -0,0 +1,357 @@
+				},
+				{
+					"label": "Hold >3secs turns on, Hold >3secs turns off",
+					"value": 6
+				}
+			]
+		},
+		"6": {
+			"label": "Key On Duration",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"8": {
+			"label": "Basic CC Integration Setting",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Key1 Responds to Basic Set and Get. No unsolicited report to Lifeline",
+					"value": 0
+				},
+				{
+					"label": "Key1 Responds to Basic Set and Get. Key1 WILL send unsolicited report to Lifeline",
+					"value": 1
+				},
+				{
+					"label": "All Keys respond to Basic Set. Basic Get ignored. No unsolicited report to Lifeline",
+					"value": 2
+				},
+				{
+					"label": "All Keys respond to Basic Set. Basic Get ignored. All keys send unsolicited report to Lifeline",
+					"value": 3
+				}
+			]
+		},
+		"10": {
+			"label": "Key 1 Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Single click to switch on/off",
+					"value": 0
+				},
+				{
+					"label": "When turned on, auto off after 'Key On Duration' seconds",
+					"value": 1
+				},
+				{
+					"label": "When turned off, auto on after 'Key On Duration' seconds",
+					"value": 2
+				},
+				{
+					"label": "Hold >3secs turns on, when released, turns off",
+					"value": 3
+				},
+				{
+					"label": "Single click for on/off, Hold >3secs on, release turns off",
+					"value": 4
+				},
+				{
+					"label": "Momentary switch: hold key turns on, off once released",
+					"value": 5
+				},
+				{
+					"label": "Hold >3secs turns on, Hold >3secs turns off",
+					"value": 6
+				}
+			]
+		},
+		"14": {
+			"label": "Disable Local Control",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "All keys are valid for local control",
+					"value": 0
+				},
+				{
+					"label": "All keys are invalid for local control",
+					"value": 1
+				}
+			]
+		},
+		"15": {
+			"label": "Disable Remote Control",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "All keys are valid for remote (gateway) control",
+					"value": 0
+				},
+				{
+					"label": "All keys are invalid for remote (gateway) control",
+					"value": 1
+				}
+			]
+		},
+		"16": {
+			"label": "Scene Respond",
```

Comment:
This label is confusing. What does it do?

Reply:
I actually have no idea - it's straight from the manual.  Would it be better to remove the parameter from the files?

![image](https://user-images.githubusercontent.com/34757455/115212868-8c0d0680-a133-11eb-8585-5e971d8c60ba.png)